<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_open_cube_prog">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(217, 217, 217, 255);
color: rgb(1, 1, 1);
border-radius: 60px;</string>
   </property>
   <property name="text">
    <string>КОНСТРУКТОРСКИЙ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open_cube_prog_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(217, 217, 217, 255);
color: rgb(1, 1, 1);
border-radius: 60px;</string>
   </property>
   <property name="text">
    <string>ПРОГРАММИРОВАНИЕ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open_cube_prog_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(217, 217, 217, 255);
color: rgb(1, 1, 1);
border-radius: 60px;</string>
   </property>
   <property name="text">
    <string>ТОПОЛОГИЯ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open_cube_prog_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>450</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(217, 217, 217, 255);
color: rgb(1, 1, 1);
border-radius: 60px;</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(217, 217, 217, 255);
color: rgb(1, 1, 1);
border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
